frontend/src/app/homepage.cljs

```diff
@@ -79,6 +93,29 @@
      (render-navigation-item "Upload" "#upload")
      (render-navigation-item "Container" "#container")]])
 
+(defn progress-width []
+  (min
+    (*
+      (float
+        (/
+          (:total_reports @backend-stats)
+    @report-target)) 100) 100)
+)
+
+(defn render-stats []
+  [:div {:id "progressbar"}
+    [:div
+      [:h5 "Are we there yet?"]
+    ]
+    [:div {
+      :id "progress"
+      :style {:width (str (progress-width) "%")}
+      }
+      [:p (progress-width) "%"]
```

Comment:
If you move the `<p>` above the `<div>`, it is easy to have the progress number always on the middle of the progress bar. Try

```clojure
(defn render-stats []
  [:div {:id "progressbar"}
   [:h5 "Are we there yet?"]
   [:div {:id "progressbar-number"}
    [:p (progress-width) "%"]]
   [:div {:id "progress"
          :style {:width (str (progress-width) "%")}}]])
```

And add the following CSS:

```css
#progressbar-number {
  position: absolute;
  margin-left: auto;
  margin-right: auto;
  margin-top: 3px;
  left: 0;
  right: 0;
  text-align: center;
}
```